app/auth.py

```
@@ -0,0 +1,360 @@
+
+def _encrypt_session(data: dict) -> str:
+    return _get_fernet().encrypt(json.dumps(data).encode()).decode("ascii")
+
+
+def _decrypt_session(token: str) -> dict | None:
+    try:
+        plaintext = _get_fernet().decrypt(
+            token.encode(), ttl=settings.session_max_age
+        )
+        return json.loads(plaintext)
+    except (InvalidToken, json.JSONDecodeError, Exception):
```

Ruff (0.15.4)

[warning] Do not catch blind exception: `Exception`

(BLE001)